cmake/projects/tommath/hunter.cmake

```diff
@@ -19,7 +19,16 @@ hunter_add_version(
     SHA1
     733e4644a0b01260830a402b457ca82064fe30cd
 )
-
+hunter_add_version(
+    PACKAGE_NAME
+    tommath
+    VERSION
+    "1.0.1"
+    URL
+    "https://github.com/t0p4/libtommath/archive/v1.0.1-p1.tar.gz"
```

Comment:
Please send your patches to https://github.com/hunter-packages/libtommath first.

I've created new branch `hunter-1.0.1` basing on `v1.0.1` tag, feel free to send pull request for review there, see:
* https://docs.hunter.sh/en/latest/creating-new/patch.html